pkg/printer/printer.go

```diff
@@ -35,6 +36,12 @@ const (
 	// https://docs.sonarqube.org/latest/analysis/generic-issue/
 	OutFormatSonarQube = "sonarqube"
 
+	// OutFormatPrometheus outputs in prometheus format
+	OutFormatPrometheus = "prometheus"
+
+	// OutFormatPushgateway send metrics to prometheus pushgateway
+	OutFormatPushgateway = "pushgateway"
```

Comment:
These are both prometheus output formats? Why do they need to be two unique ones? Also, please include links here to prometheus format like the other formats provide